snap-jp2-reader/src/main/java/org/esa/snap/jp2/reader/metadata/Jp2XmlMetadata.java

```diff
@@ -134,15 +134,18 @@ public String getProductDescription() {
         return null;
     }
 
+    /**
```

Comment:
Don't use api-doc comments for code comments. Simply start with /* instead with /**

Same for the other 3 comments

Reply:
Code refactoring done.